src/kernel/normalizer_extension.cpp

```diff
@@ -54,7 +54,7 @@ class comp_normalizer_extension : public normalizer_extension {
     }
 };
 
-std::unique_ptr<normalizer_extension> compose(std::unique_ptr<normalizer_extension> && ext1, std::unique_ptr<normalizer_extension> && ext2) {
+std::unique_ptr<normalizer_extension> compose_ext(std::unique_ptr<normalizer_extension> && ext1, std::unique_ptr<normalizer_extension> && ext2) {
     return std::unique_ptr<normalizer_extension>(new comp_normalizer_extension(std::move(ext1), std::move(ext2)));
```

Comment:
Is `compose` a reserved keyword on VS?

Reply:
No, but it clashes with the other compose() functions. VS gets confused in some files and can't infer which one to call, so I had to change the name.